types/aws-lambda/common/api-gateway.d.ts

```diff
@@ -28,25 +35,27 @@ export interface APIGatewayEventRequestContextWithAuthorizer<TAuthorizerContext>
     // This lets us allow parameterizing the authorizer for proxy events that know what authorizer
     // context values they have.
     authorizer: TAuthorizerContext;
-    connectedAt?: number;
-    connectionId?: string;
+    awsEndpointRequestId?: string;
     domainName?: string;
     domainPrefix?: string;
+    error?: APIGatewayError;
     eventType?: string;
     extendedRequestId?: string;
-    protocol: string;
     httpMethod: string;
-    identity: APIGatewayEventIdentity;
-    messageDirection?: string;
-    messageId?: string | null;
+    identity?: APIGatewayEventIdentity;
```

Comment:
This is my main concern: when do you see this not defined? Being a back-incompatible change, I'm concerned that this will break far more people than it will save, if it's just the AWS console tester being crappy again.

Reply:
reverted. Not sure why I did that.